validator/sawtooth_validator/journal/genesis.py

```diff
@@ -212,11 +217,17 @@ def _get_block_publisher(self, state_hash):
         """
         state_view = self._state_view_factory.create_view(state_hash)
         try:
+            class BatchPublisher(object):
+                def send(self, transactions):
+                    raise InvalidGenesisConsensusError(
```

Comment:
Is there a way we can document this?  It's not entirely clear from the source alone.